src/api/currencies_ticker.ts

```diff
@@ -54,7 +55,9 @@ const currenciesTicker = async (
     key
   };
 
-  return fetchJSON(`${CURRENCIES_TICKER_URL}?${objToUrlParams(objParams)}`);
+  return fetchJSON(
+    `${apiBase}${CURRENCIES_TICKER_PATH}?${objToUrlParams(objParams)}`
```

Comment:
Can this be a getter of `NOMICS_API_BASE` from `index.ts` rather than passing the URL around?

Reply:
Yeah I agree. I didn't like this much either. The only other way that comes to mind is to turn this variable into a static property and then reference it that way inside the ticker functions instead of getting it via parameter.